packages/combo-box/src/vaadin-combo-box-data-provider-mixin.js

```diff
@@ -81,20 +70,38 @@ export const ComboBoxDataProviderMixin = (superClass) =>
       ];
     }
 
+    constructor() {
+      super();
+
+      /**
+       * @type {DataProviderController}
+       * @private
+       */
+      this.__dataProviderController = new DataProviderController(this, {
+        size: this.size,
+        pageSize: this.pageSize,
+        placeholder: new ComboBoxPlaceholder(),
+        isPlaceholder: (item) => item instanceof ComboBoxPlaceholder,
```

Comment:
Seems like this line is untested (all the tests still pass when I remove it).

Reply:
This line ensures backward compatibility if someone, like the Flow connector, directly manipulates filteredItems and adds their own instances of ComboBoxPlaceholder. The Flow connector should ideally reuse the placeholder instance that is already available in the connector. However, there could also be workarounds that add their own instances of ComboBoxPlaceholder. I guess I could add a test imitating such a workaround.

Reply:
Thanks for clarification. I think we can keep this logic as is then.